### PR TITLE
Loading arbitrary contents

### PR DESCRIPTION
**Pull request by rgbkrk**
Switch us to a Contents Switcher Component.

On arbitrary, unknown contents, this shows "Loading..." for now. My next PR is going to have the directory bits (and is based on this branch so this will need to be merged first).

### CODE REVIEW

File: packages/core/src/components/contents/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    // TODO: This could delegate to the component below to render based on
+    //       the passed in content ref
+    //       For now, notebook app still relies on currentContentRef directly
+    contentRef: selectors.currentContentRef(state),
+    currentContent: selectors.currentContent(state)
+    */
+  };
+};
+
+const mapDispatchToProps = (state: *, ownProps: *): * => {
```
Comment (theengineear) [inline packages/core/src/components/contents/index.js]: ⚡️

File: packages/core/src/components/contents/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  return {};
+};
+
+class Contents extends React.PureComponent<*, *> {
+  render() {
+    switch (this.props.contentType) {
```
Comment (theengineear) [inline packages/core/src/components/contents/index.js:31]: 👌

File: packages/core/src/components/contents/index.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      case "notebook":
+        return (
+          <React.Fragment>
+            <TitleBar />
+            <NotebookMenu />
```
Comment (theengineear) [inline packages/core/src/components/contents/index.js:36]: 👌 solid!

**Review by theengineear — APPROVED** [review]
Looking awesome! I'm surprised it only took that many lines of code changes!

File: .flowconfig [hunk 4]
```diff
@@ -15,6 +15,7 @@ emoji=true
 
 [ignore]
 .*/node_modules/eslint-plugin-jsx-a11y/src/.*
+.*/node_modules/webpack-cli/**/*.json
```
Comment (rgbkrk) [inline .flowconfig:18]: xref for why I did this: https://github.com/webpack/webpack-cli/pull/349

**Pull request merged**